packages/remix/README.md

```diff
@@ -0,0 +1,5 @@
+# @clerk/clerk-remix
+
+Clerk SDK for authentication and user management in your Remix application.
+
+For instructions on how to get started go to [Clerk.dev official Next.js documentation](https://docs.clerk.dev/get-started/remix).
```

Comment:
:upside_down_face: The link text mentions Next.js :smile:

Reply:
❓ Can we also add a usage example in the README.md? We are in the process of enriching all our README.md files in the monorepo.

Reply:
Oops. Thanks team, I'll create a sample repo today and update the readme accordingly